lib/client.js

```diff
@@ -1449,6 +1450,37 @@ MatrixClient.prototype.getCurrentUploads = function() {
 };
 
 /**
+ * Get a preview of the given URL as OpenGraph metadata.
+ * Caches results to prevent hammering the server.
+ * @param {string} url The URL to get preview data for
+ * @param {Number} ts The timestamp (ms since epoch) for querying
+ * historical preview data.
```

Comment:
this is a bit vague. Not everyone should be expected to read the react-sdk source to figure out what it means. Can you clarify how it is meant to be used?